.travis.yml

```diff
@@ -37,11 +36,11 @@ matrix:
       sudo: required
       dist: trusty
       env:
-        - SWIFT_VERSION=4.0.3
+        - SWIFT_VERSION=4.1
 install:
   - if [[ "$TRAVIS_OS_NAME" == "osx"   ]]; then ./script/travis-install-macos; fi
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then ./script/travis-install-linux; fi
-  - if [[ "$PODSPEC" ]]; then rvm system; sudo gem install bundler; bundle install; fi
+  - if [[ "$PODSPEC" ]]; then gem install bundler; bundle install; fi
```

Comment:
See https://github.com/Quick/Quick/pull/777#discussion_r180744388 for the details.